--- node_modules/ast-types/lib/def/core.js	2022-04-29 20:15:23.000000000 +0200
+++ node_modules/ast-types/lib/def/core.js	2022-05-08 23:47:54.000000000 +0200
@@ -262,7 +262,7 @@
     def("Literal")
         .bases("Expression")
         .build("value")
-        .field("value", or(String, Boolean, null, Number, RegExp, BigInt));
+        .field("value", or(String, Boolean, null, Number, RegExp));
     // Abstract (non-buildable) comment supertype. Not a Node.
     def("Comment")
         .bases("Printable")
