--- node_modules/compromise/src/1-one/tokenize/methods/01-sentences/is-sentence.js	2024-08-30 13:59:03
+++ node_modules/compromise/src/1-one/tokenize/methods/01-sentences/is-sentence.js	2024-08-30 16:51:17
@@ -1,6 +1,12 @@
+const xretest = re => {
+  re = new Zotero.Utilities.XRegExp(re)
+  return {
+    test: s => Zotero.Utilities.XRegExp.test(s, re)
+  }
+}
 const isAcronym = /[ .][A-Z]\.? *$/i //asci - 'n.s.a.'
 const hasEllipse = /(?:\u2026|\.{2,}) *$/ // '...'
-const hasLetter = /\p{L}/u
+const hasLetter = xretest('\\p{L}')
 const hasPeriod = /\. *$/
 const leadInit = /^[A-Z]\. $/ // "W. Kensington"
 
