--- node_modules/compromise/src/1-one/tokenize/methods/03-whitespace/tokenize.js	2024-08-30 16:21:05
+++ node_modules/compromise/src/1-one/tokenize/methods/03-whitespace/tokenize.js	2024-08-30 16:31:39
@@ -1,8 +1,14 @@
 //all punctuation marks, from https://en.wikipedia.org/wiki/Punctuation
 
+const xretest = re => {
+  re = new Zotero.Utilities.XRegExp(re)
+  return {
+    test: s => Zotero.Utilities.XRegExp.test(s, re)
+  }
+}
 //we have slightly different rules for start/end - like #hashtags.
-const isLetter = /\p{Letter}/u
-const isNumber = /[\p{Number}\p{Currency_Symbol}]/u
+const isLetter = xretest('\\p{Letter}')
+const isNumber = xretest('[\\p{Number}\\p{Currency_Symbol}]')
 const hasAcronym = /^[a-z]\.([a-z]\.)+/i
 const chillin = /[sn]['’]$/
 
