--- node_modules/kuroshiro/src/core.js	2018-10-19 10:25:23.000000000 +0200
+++ node_modules/kuroshiro/src/core.js	2018-10-24 15:53:21.159782682 +0200
@@ -67,9 +67,9 @@
      * @param {string} [options.romajiSystem="hepburn"] Romanization System ["nippon"|"passport"|"hepburn"]
      * @param {string} [options.delimiter_start="("] Delimiter(Start)
      * @param {string} [options.delimiter_end=")"] Delimiter(End)
-     * @returns {Promise} Promise object represents the result of conversion
+     * @returns {string} result of conversion
      */
-    async convert(str, options) {
+    convert(str, options) {
         options = options || {};
         options.to = options.to || "hiragana";
         options.mode = options.mode || "normal";
@@ -91,7 +91,7 @@
             throw new Error("Invalid Romanization System.");
         }
 
-        const rawTokens = await this._analyzer.parse(str);
+        const rawTokens = this._analyzer.parse(str);
         const tokens = patchTokens(rawTokens);
 
         if (options.mode === "normal" || options.mode === "spaced") {
