diff --git a/node_modules/kuromoji/src/loader/BrowserDictionaryLoader.js b/node_modules/kuromoji/src/loader/BrowserDictionaryLoader.js
index 04bfdcd..55ecff5 100644
--- a/node_modules/kuromoji/src/loader/BrowserDictionaryLoader.js
+++ b/node_modules/kuromoji/src/loader/BrowserDictionaryLoader.js
@@ -17,11 +17,10 @@
 
 "use strict";
 
-var zlib = require("zlibjs/bin/gunzip.min.js");
 var DictionaryLoader = require("./DictionaryLoader");
 
 /**
- * BrowserDictionaryLoader inherits DictionaryLoader, using jQuery XHR for download
+ * BrowserDictionaryLoader inherits DictionaryLoader, using fetch for download
  * @param {string} dic_path Dictionary path
  * @constructor
  */
@@ -32,29 +31,24 @@ function BrowserDictionaryLoader(dic_path) {
 BrowserDictionaryLoader.prototype = Object.create(DictionaryLoader.prototype);
 
 /**
- * Utility function to load gzipped dictionary
+ * Utility function to load (already unzipped) dictionary
  * @param {string} url Dictionary URL
  * @param {BrowserDictionaryLoader~onLoad} callback Callback function
  */
-BrowserDictionaryLoader.prototype.loadArrayBuffer = function (url, callback) {
-    var xhr = new XMLHttpRequest();
-    xhr.open("GET", url, true);
-    xhr.responseType = "arraybuffer";
-    xhr.onload = function () {
-        if (this.status > 0 && this.status !== 200) {
-            callback(xhr.statusText, null);
+BrowserDictionaryLoader.prototype.loadArrayBuffer = async function (url, callback) {
+    url = `chrome://zotero-better-bibtex/content/resource/kuromoji/${url.replace(/.*[\\/]/, '').replace(/\.gz$/, '')}`
+    try {
+        var response = await fetch(url);
+        if (!response.ok) {
+            callback(response.statusText, null);
             return;
         }
-        var arraybuffer = this.response;
-
-        var gz = new zlib.Zlib.Gunzip(new Uint8Array(arraybuffer));
-        var typed_array = gz.decompress();
-        callback(null, typed_array.buffer);
-    };
-    xhr.onerror = function (err) {
+        var arraybuffer = await response.arrayBuffer();
+        callback(null, arraybuffer);
+    }
+    catch (err) {
         callback(err, null);
-    };
-    xhr.send();
+    }
 };
 
 /**
