diff --git a/node_modules/zotero-types/types/gecko.d.ts b/node_modules/zotero-types/types/gecko.d.ts
index d675f24..c0d00fb 100644
--- a/node_modules/zotero-types/types/gecko.d.ts
+++ b/node_modules/zotero-types/types/gecko.d.ts
@@ -27,8 +27,6 @@ interface DOMStringMap {
   [name: string]: string | undefined;
 }
 
-interface EventListenerOrEventListenerObject extends EventListener {}
-
 interface EventSourceEventMap {
   /**
    * Returns an object that provides methods and properties for drawing and manipulating images and graphics on a canvas element in a document. A context object includes information about colors, line widths, fonts, and other graphic parameters that can be drawn on a canvas.
