--- node_modules/lokijs/src/lokijs.js	2018-05-28 10:13:19.000000000 +0200
+++ node_modules/lokijs/src/lokijs+logging.js	2018-12-01 11:51:24.096263720 +0100
@@ -4,6 +4,12 @@
  *
  * A lightweight document oriented javascript database
  */
+
+const consoleService = Components.classes["@mozilla.org/consoleservice;1"].getService(Components.interfaces.nsIConsoleService);
+function logMessage(msg) {
+  consoleService.logStringMessage("LokiJS: " + msg)
+}
+
 (function (root, factory) {
   if (typeof define === 'function' && define.amd) {
     // AMD
@@ -6140,7 +6146,10 @@
         throw new TypeError('Passed id is not an integer');
       }
 
+      logMessage(`Looking for ${id} in ${data} between ${min} and ${max}`)
       while (data[min] < data[max]) {
+        logMessage(`Looking for ${id} in ${data} between ${min} and ${max}`)
+
         mid = (min + max) >> 1;
 
         if (data[mid] < id) {
