--- build/content/common.js	2018-09-04 22:29:28.163699671 +0200
+++ build/content/common.js.patched	2018-09-04 22:33:21.142503181 +0200
@@ -64,8 +64,10 @@
 /******/ 	// The require function
 /******/ 	function __webpack_require__(moduleId) {
 /******/
+/******/ 		Zotero.debug('BBT: webpack: require ' + moduleId);
 /******/ 		// Check if module is in cache
 /******/ 		if(installedModules[moduleId]) {
+/******/ 			Zotero.debug('BBT: webpack: fetched ' + moduleId);
 /******/ 			return installedModules[moduleId].exports;
 /******/ 		}
 /******/ 		// Create a new module (and put it into the cache)
@@ -82,6 +84,7 @@
 /******/ 		module.l = true;
 /******/
 /******/ 		// Return the exports of the module
+/******/ 		Zotero.debug('BBT: webpack: cached ' + moduleId);
 /******/ 		return module.exports;
 /******/ 	}
 /******/
@@ -108750,4 +108753,4 @@
 
 /***/ })
 
-/******/ });
\ No newline at end of file
+/******/ });
